acceptance/bundle/telemetry/deploy-compute-type/script

```diff
@@ -0,0 +1,8 @@
+trace $CLI bundle deploy -t one
+trace $CLI bundle deploy -t two
+trace $CLI bundle deploy -t three
+trace $CLI bundle deploy -t four
```

Comment:
The test was killed with timeout on CI: https://github.com/databricks/cli/actions/runs/15022764995/job/42215860250

You can raise timeout or break it into individual tests instead.